gapic/src/main/com/google/gapid/perfetto/models/ArgSet.java

```diff
@@ -30,11 +30,11 @@
   private static final String ARGS_QUERY =
       "select key, int_value, real_value, string_value from args where arg_set_id = %d";
 
-  public static final ArgSet EMPTY = new ArgSet(ImmutableMap.of());
+  public static final ArgSet EMPTY = new ArgSet(new LinkedHashMap());
```

Comment:
Adding the names to the args table effectively nullify the point of putting the debug names into the table.  If GAPID were to add custom UI logic to change how the properties are displayed, the UI code will be the same as before.

Reply:
I added it to the args table because we wanted to show the markers in the properties pane. If not, then this can be avoided. Adding some random field to the properties pane without them being in the argset will be a mess of a code.